cime_config/namelist_definition_drv.xml

```diff
@@ -891,7 +891,7 @@
       default: ocn
     </desc>
     <values>
-      <value>ogrid</value>
+      <value>xgrid</value>
```

Comment:
I don't think this should be part of this PR.

Reply:
You are correct - I removed it in my sandbox, but forgot to push that back.